### ISSUE TITLE
Test WSL support for inspector protocol

### ISSUE THREAD

**Issue by weinand** [Microsoft/vscode#37168]
Test for https://github.com/Microsoft/vscode-node-debug2/pull/142:

Complexity: 3

- [x] Windows - @weinand 

Setup:
- install WSL on Windows 10.
- in Windows create a node.js test program and a launch config
- add a `"useWSL": "true"` attribute to the launch config
- launch the debug session

Verify for the "inspector" protocol that:
- the node.js version detection finds the correct node.js version on the linux side
- the debug session launches correctly and that you can step, set and hit breakpoints and view sources

/cc @roblourens I've copied this from the September test item for "legacy". Please adapt if needed (and remove this comment).

**Comment by weinand** [comment]
I verified hat debugging node on the linux side of WSL works fine.

However, I noticed one difference in the echoed command line.

For the legacy protocol:
![2017-10-31_11-36-18](https://user-images.githubusercontent.com/1898161/32220147-fefe7f0a-be30-11e7-96b7-d1d46e631c9b.png)

inspector protocol:
![2017-10-31_11-36-58](https://user-images.githubusercontent.com/1898161/32220173-13965014-be31-11e7-98ca-ae42d4c7572f.png)

I'm not sure whether we really want to show the bash wrapper of the node command line.
@bzoz @roblourens opinions?
